keybase/platform_windows.go

```diff
@@ -175,6 +159,26 @@ func checkRebootPending(wow64 bool, log Log) (bool, error) {
 		}
 	}
 
+	// Check for a Dokan reboot pending
+	k2, err := registry.OpenKey(registry.LOCAL_MACHINE, "SYSTEM\\CurrentControlSet\\Control\\Session Manager", access)
+	if err != nil {
+		log.Errorf("Error opening Session Manager key: %s", err)
+		return false, err
+	}
+	defer util.Close(k2)
+	vals, _, err := k2.GetStringsValue("PendingFileRenameOperations")
+	if err != nil {
+		// This is normal if no reboot is pending
+		log.Errorf("Error getting PendingFileRenameOperations: %s", err)
+	} else {
+		for _, val := range vals {
+			if strings.Contains(strings.ToLower(val), "dokan") {
```

Comment:
The worst that will happen is that the installer will not be launched silently in that case, and the updater prompt will show instead.